test/geo.test.js

```diff
@@ -0,0 +1,117 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+require('./init.js');
+var _ = require('lodash');
+var async = require('async');
+var should = require('should');
+var url = require('url');
+
+var db, sampleData;
+
+describe('cloudant geo', function() {
+  describe('viewDocs', function(done) {
+    before(function(done) {
+      db = global.getDataSource();
+      var connector = db.connector;
+
+      db.once('connected', function(err) {
+        async.series([insertSampleData, insertViewDdoc], done);
+      });
+
+      function insertSampleData(cb) {
+        sampleData = generateSamples();
+        connector[connector.name].use(connector.getDbName(connector))
```

Comment:
nitpick: 
`var driverInstance = connector[connector.name].use(connector.getDbName(connector))`

then you can call `driverInstance.bulk`, `driverInstance.insert`